compiler/pipes/parse-and-apply-phpdoc.cpp

```diff
@@ -361,60 +361,90 @@ class ParseAndApplyPHPDoc {
       }
     }
   }
+};
 
+// Inspects @var above all fields and converts them to type hints
+// Also analyzes @kphp-serializable and other tags above the class itself
+class ParseAndApplyPhpDocForClass {
+  ClassPtr klass;
```

Comment:
Maybe it would be better to place it below `private:`?

Reply:
For me, it's better when members are above implementation. The same for kbasename